apiary.apib

```diff
@@ -2131,6 +2132,7 @@ Draft a new invoice.
                         + percentage
                 + description: `winter promotion` (string, required)
         + note: `Invoice comments` (string, optional)
+        + expected_payment_method (ExpectedPaymentMethod, optional)
```

Comment:
PP: make it nullable in the .draft as well

Reply:
then should we make all optional fields in the draft nullable?

Reply:
B: the code doesn't accept null